Makefile

```diff
@@ -15,7 +19,7 @@ cppcheck: cppcheck-build-dir
 
 infer:
 	python3 setup.py build_ext -if -U NDEBUG | compiledb
-	@infer --fail-on-issue --compilation-database compile_commands.json
+	@infer --fail-on-issue --compilation-database compile_commands.json --report-blacklist-path-regex "hpy/debug/src/debug_ctx.c"
```

Comment:
Should we perhaps leave a comment on why `debug_ctx.c` had to be blacklisted?

Reply:
good idea. The full explanation is in the log message of cd8cd6e, I added a comment which redirects to it